Flutter/calendar/appointments.md

```diff
@@ -202,7 +202,7 @@ class MeetingDataSource extends CalendarDataSource {
 You can schedule meetings for a day by setting `From` and `To` of Meeting class. Create meetings of type `List<Meeting>` and assign those appointments collection Meetings to the [appointments](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/CalendarDataSource/appointments.html) property of `CalendarDataSource`.
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="7 16 17 18 19 20 21 22 23 24 26" %}
```

Comment:
https://help.syncfusion.com/flutter/calendar/appointments

Not highlighted correctly

Reply:
Highlighted the document properly

Reply:
Not highlighted correctly

Reply:
Highlighted correctly.